tests/lua/machine-bind.lua

```diff
@@ -540,13 +540,13 @@ end)
 
 do_test("should error if target mcycle is smaller than current mcycle", function(machine)
     machine:write_reg("mcycle", MAX_MCYCLE)
-    assert(machine:read_mcycle() == MAX_MCYCLE)
+    assert(machine:read_reg("mcycle") == MAX_MCYCLE)
```

Comment:
We still have `read_mcycle`, no need to change to `read_reg` in this case and all the ones below.